src/supertux/levelset_screen.cpp

```diff
@@ -27,6 +27,8 @@
 #include "util/file_system.hpp"
 #include "util/log.hpp"
 
+// TODO: wtf?????????
+
```

Comment:
You'll fix this anyway, not needed.

Reply:
Ha, this was from my cherry picked commit, my bad :)